plugin/mouse/mouse.talon

```diff
@@ -86,10 +86,10 @@ wheel tiny [down]: user.mouse_scroll_down(0.2)
 wheel tiny [down] here:
     user.mouse_move_center_active_window()
     user.mouse_scroll_down(0.2)
-wheel downer: user.mouse_scroll_down_continuous()
-wheel downer here:
+wheel downer [<number_small>]: user.mouse_scroll_down_continuous(number_small or 10)
```

Comment:
pass none instead of 10 and make it optional in the actions

Reply:
As none can only be passed in beta, do you think it would be better to just have a separate version of the command for the argument or to do something like use "" or -1 to denote none? I personally think a separate version would be easier to read, but I do not have a strong opinion either way.

Reply:
Unfortunately making separate commands is the best way for now I think.

Reply:
I made the change, but there is a bug I should fix before merge.